src/runtime/basic-runtime.js

```diff
@@ -445,6 +445,10 @@ export class ConfiguredBasicRuntime {
     return promise.then((response) => {
       const jwt = response['jwt'];
       if (jwt) {
+        // If entitlements are returned, close the subscription/contribution offers iframe
+        this.configuredClassicRuntime_.closeDialog();
```

Comment:
Is it possible to add test coverage for this?

Reply:
Done

Reply:
Nice work!